crates/bevy_app/src/app.rs

```diff
@@ -23,12 +23,9 @@ bevy_utils::define_label!(
 );
 
 #[allow(clippy::needless_doctest_main)]
-/// A container of app logic and data.
+/// An ECS application that wraps a [`World`], a runner function, and a sub-[`App`] collection.
```

Comment:
I don't like these changes; it ties the docs very closely to the implementation details without providing higher-level context for an important struct.

Reply:
The fields are public / have public accessors, so I wouldn't consider them implementation details. What if we keep the high level explanation, then elaborate on the specifics?